inference-engine/src/vpu/graph_transformer/include/vpu/private_plugin_config.hpp

```diff
@@ -36,6 +36,13 @@ DECLARE_VPU_CONFIG(MYRIAD_PER_LAYER);
 DECLARE_VPU_CONFIG(MYRIAD_PER_STAGE);
 
 DECLARE_VPU_CONFIG(MYRIAD_ENABLE_MEMORY_TYPES_ANNOTATION);
+
+/**
+ * @brief Used to disable analyzeWeightableLayers pass in cases where
+ * weights scaling leads to poor accuracy. Default = "NO"
+ */
+DECLARE_VPU_CONFIG(MYRIAD_DISABLE_WEIGHTS_ANALYSIS);
```

Comment:
It is just a nitpick, but I'd prefer to add new configs with `ENABLE_` logic. So in this case, `Default` will be "YES".

Reply:
Alright. I've renamed it.